ganalytics.php

```diff
@@ -64,14 +64,28 @@ private function getOptOutConfiguration(){
         return $optout_config;
     }
 
+    /**
+     * Return the Google Analytics Tracking Code
+     * @param string $objectName Global variable name for the GA object
+     * @return string
+     */
+    private function getGa4TrackingCode($objectName)
+    {
+        $code = $"window.dataLayer = window.dataLayer || [];\n".
+              "function gtag(){dataLayer.push(arguments);}\n".
+              "gtag('js', new Date());\n";
+
+        return $code;
+    }
+
     /**
      * Return the Google Analytics Tracking Code
      * @param string $scriptName Name of the GA script library
      * @param string $objectName Global variable name for the GA object
      * @param bool $async Determine if the GA script should be loaded and executed asynchronously
      * @return string
      */
-    private function getTrackingCode($scriptName, $objectName, $async=false)
+    private function getOldTrackingCode($scriptName, $objectName, $async=false)
```

Comment:
**Chore:** It's really a bad idea to name things `old` or `new`. Please change it to `getUATrackingCode()` for eg. Otherwise `getGa4TrackingCode()` could be named `getNewTrackingCode()` - I hope you see how it might lead to a huge inconvenience in the future 🙂